src/features/profile/pages/ConfirmChangeEmail/ConfirmChangeEmail.web.test.tsx

```diff
@@ -28,9 +23,11 @@ describe('<ConfirmChangeEmail />', () => {
     it('should not have basic accessibility issues', async () => {
       const { container } = render(reactQueryProviderHOC(<ConfirmChangeEmail />))
 
-      const results = await checkAccessibilityFor(container)
+      await act(async () => {
+        const results = await checkAccessibilityFor(container)
 
-      expect(results).toHaveNoViolations()
+        expect(results).toHaveNoViolations()
+      })
```

Comment:
mélanger une assertion dans un act me parait etre une mauvaise chose

serait possible de faire ça ?

```suggestion
      const results = await act(async () => checkAccessibilityFor(container))

      expect(results).toHaveNoViolations()
```

si c'est pas possible, est-ce qu'on peut tenter ça

```suggestion
      let results
      await act(async () => {
        results = await checkAccessibilityFor(container)
      })
      
      expect(results).toHaveNoViolations()
```

Reply:
La deuxième proposition avec le let fonctionne (mais pas la première de sortir le expect)